project_testing/housing_characteristics/Mechanical Ventilation.tsv

```diff
@@ -1,2 +1,2 @@
-Option=None	Option=Exhaust, 2010 ASHRAE 62.2	Option=Exhaust, 50%, 2013 ASHRAE 62.2	Option=Supply, 2010 ASHRAE 62.2	Option=HRV, 60%, 2010 ASHRAE 62.2	Option=ERV, 72%, 2010 ASHRAE 62.2	sampling_probability
-0.166667	0.166667	0.166667	0.166667	0.166667	0.166667	1.000000
+Option=None	Option=Exhaust	Option=Supply	"Option=HRV, 60%"	"Option=ERV, 72%"	sampling_probability
```

Comment:
Going to need to add these changes to resstock-estimation.

Reply:
Yep, that's what I said right above.